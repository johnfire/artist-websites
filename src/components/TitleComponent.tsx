import React from "react";
import { useTranslations, useFormatter } from "next-intl";

const TitleComponent = () => {
  console.log("TC5 title ");
  const t = useTranslations("front_page");
  // const format = useFormatter(); // use later

  return (
    <>
      <div>&nbsp;&nbsp;{t("title")}&nbsp;&nbsp;</div>
    </>
  );
};

export default TitleComponent;
