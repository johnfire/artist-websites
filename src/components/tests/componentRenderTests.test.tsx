import React from "react";
import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";
import TitleComponent from "../TitleComponent";

describe("Components: some basic tests.", () => {
  it("adds two and two", () => {
    expect(2 + 2).toBe(4);
  });

  it("first renderer test ", () => {
    const component = renderer.create(<TitleComponent />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
