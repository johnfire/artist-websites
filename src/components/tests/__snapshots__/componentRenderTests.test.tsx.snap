// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components: some basic tests. first renderer test  1`] = `
<div>
  Welcome to the Artists "make your website" application
</div>
`;
